src/IndexingWrapper.cpp

```diff
@@ -10,88 +10,94 @@
 using namespace std;
 
 int
-rowCount(Rcpp::DataFrame & dataframe)
-{
+  rowCount(Rcpp::DataFrame & dataframe)
```

Comment:
Do you want things formatting like this? It's pretty unusual, and I haven't seen other C++ laid out like that.